ignite/internal/plugin/consumer.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 const (
-	PluginConsumerVersion = "consumer/v0.1.0"
+	PluginConsumerVersion = "36532a375ecec4a228dc9b320c1576a2a1fafd34"
```

Comment:
This needs https://github.com/ignite/apps/pull/89 merged first, then I can tag and we can use consumer/v0.2.0 here.